examples/credential_offer_multiple_credentials.json

```diff
@@ -10,7 +10,7 @@
          "tx_code" : {
             "length": 4,
             "input_mode" : "numeric",
-            "description": "Please provide the PIN which was sent via e-mail"
+            "description": "Please provide the Transaction Code which was sent via e-mail"
```

Comment:
The Issuer doesn't necessarily need to call it Transaction Code in his communciation to the Holder, but PIN is probably confusing here. What about OTP?

Reply:
True, transaction code may sound a bit too technical for end-users, I would also prefer OTP or rather the written word "one-time password".

Reply:
I changed it to "one-time code", I think that is the clearest notation in this case. What do you think? @Sakurann @paulbastian